modules/nw-sriov-configuring-device.adoc

```diff
@@ -55,18 +55,18 @@ spec:
 CNI plug-in and device plug-in will be only deployed on selected nodes.
-<5> Specify an integer value between `0` and `99`. A larger number gets lower priority, so a priority of `99` is lower than a priority of `10`.
-<6> Specify a value for the maximum transmission unit (MTU) of the virtual function. The value for MTU must be in the range from `1` to `9000`. If you do not need to specify the MTU, specify a value of `''`. 
+<5> Optional. Specify an integer value between `0` and `99`. A larger number gets lower priority, so a priority of `99` is lower than a priority of `10`. The default value is `99`.
+<6> Optional. Specify a value for the maximum transmission unit (MTU) of the virtual function. The maximum MTU value can vary for different NIC models.
 <7> Specify the number of the virtual functions (VF) to create for the SR-IOV physical network device. For an Intel Network Interface Card (NIC), the number of VFs cannot be larger than the total VFs supported by the device. For a Mellanox NIC, the number of VFs cannot be larger than `128`.
 <8> The `nicSelector` mapping selects the Ethernet device for the Operator to configure. You do not need to specify values for all the parameters. It is recommended to identify the Ethernet adapter with enough precision to minimize the possibility of selecting an Ethernet device unintentionally.
 If you specify `rootDevices`, you must also specify a value for `vendor`, `deviceID`, or `pfName`.
 If you specify both `pfName` and `rootDevices` at the same time, ensure that they point to an identical device.
-<9> Specify the vendor hex code of the SR-IOV network device. The only allowed values are either `8086` or `15b3`.
-<10> Specify the device hex code of SR-IOV network device. The only allowed values are `158b`, `1015`, `1017`.
-<11> The parameter accepts an array of one or more physical function (PF) names for the Ethernet device.
+<9> Optional. Specify the vendor hex code of the SR-IOV network device. The only allowed values are either `8086` or `15b3`.
+<10> Optional. Specify the device hex code of SR-IOV network device. The only allowed values are `158b`, `1015`, `1017`.
+<11> Optional. The parameter accepts an array of one or more physical function (PF) names for the Ethernet device.
 <12> The parameter accepts an array of one or more PCI bus addresses for the physical function of the Ethernet device. Provide the address in the following format: `0000:02:00.1`.
-<13> Specify the driver type for the virtual functions. You can specify one of the following values: `netdevice` or `vfio-pci`. The default value is `netdevice`.
-<14> Specify whether to enable RDMA mode. The default value is `false`. Only RDMA over Converged Ethernet (RoCE) mode is supported on Mellanox Ethernet adapters.
+<13> Optional. Specify the driver type for the virtual functions. You can specify one of the following values: `netdevice` or `vfio-pci`. The default value is `netdevice`.
```

Comment:
Shall we add a note here: `For mellanox card to work in dpdk mode, use netdevice driver type`